Source/WebGPU/WebGPU/CommandEncoder.swift

```diff
@@ -1896,15 +1894,13 @@ extension WebGPU.CommandEncoder {
             guard ensureBlitCommandEncoder() != nil else {
                 return
             }
-            var counterSampleBuffer: MTLCounterSampleBuffer? = nil
-            counterSampleBuffer = unsafe querySet.counterSampleBufferWithOffset().first
-            let counterSampleBufferOffset = unsafe querySet.counterSampleBufferWithOffset().second
-            guard let counterSampleBuffer else {
+            let counterSampleBuffer = querySet.counterSampleBufferWithOffset()
+            guard counterSampleBuffer.buffer != nil else {
```

Comment:
guard let here too

Reply:
but it wouldn’t be unused? line 1902 would use it

Reply:
Yeah, that's fair, I can change that line to match.